cmd/root.go

```diff
@@ -75,6 +92,12 @@ func init() {
 
 	RootCmd.PersistentFlags().StringSliceVarP(&exclude, "exclude", "e", []string{}, "List of resources to not import, this names are the ones on TF (ex: aws_instance). If not set then means that none the resources will be excluded")
 	_ = viper.BindPFlag("exclude", RootCmd.PersistentFlags().Lookup("exclude"))
+
+	RootCmd.PersistentFlags().BoolP("verbose", "v", false, "Activate the verbose mode")
+	_ = viper.BindPFlag("verbose", RootCmd.PersistentFlags().Lookup("verbose"))
+
+	RootCmd.PersistentFlags().BoolP("debug", "d", false, "Activate the debug mode wich includes TF logs via TF_LOG configuration https://www.terraform.io/docs/internals/debugging.html")
```

Comment:
I'd update the message to be more explicit about the content of TF_LOG, which could be missleading for beginner. 
I'd put something like: `TF_LOG=TRACE|DEBUG|INFO|WARN|ERROR` which is more explicit, while still keeping the link to terraform.io.